cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/machineconfig.yaml

```diff
@@ -0,0 +1,26 @@
+# Generated by Butane; do not edit
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
```

Comment:
just wanted to double check that we are only doing this for the controllers

Reply:
This is for the infra cluster, which at the moment only has controllers.